spec/support/data_export_shared_example.rb

```diff
@@ -9,10 +9,4 @@
 
     expect(exported_columns).to match_array(documented_columns)
   end
-
-  it 'doesn’t shadow any common columns' do
-    shadowed_columns = DataSetDocumentation.shadowed_common_columns(described_class)
-
-    expect(shadowed_columns.count).to eq(0)
-  end
```

Comment:
If there are any shadowed columns the class will raise an exception during the first test. 

It would still be nice to add a test in if we could, but I'm struggling to see how you'd do it. Feedback very welcome!

Reply:
Yea this is tricky. I guess you could test the DataSetDocumentation service directly?

Something like

```
  it 'raises an error if there are shared columns' do
    common_columns = ['provider_code']
    custom_columns = ['provider_code']

    expect(DataSetDocumentation.check_for_shadowed_columns(common_columns, 
      custom_columns)).raise_error Errno::ENOENT 
  end
```

Reply:
The first test should catch it anyway though so i think that's fine.

Reply:
Neat solution David, thanks! But I also agree that we probably don't need it here.